src/utils/string.js

```diff
@@ -241,3 +241,7 @@ export function ensureRegex(strOrRegex, exact = true) {
 export function nlToBr(value) {
   return escapeHtml(value || '').replace(/(\r\n|\r|\n)/g, '<br/>\n');
 }
+
+export function stringifyWithWhiteSpace(obj) {
+  return `${ JSON.stringify(obj, undefined, 2) }\n`;
+}
```

Comment:
This file is from dashboard; adding to it might not be a great idea (for when we need to update it).

Reply:
Right, and that solves the typescript problem with one of the modules.